repose-aggregator/components/filters/rackspace-identity-basic-auth/src/main/scala/org/openrepose/filters/rackspaceidentitybasicauth/RackspaceIdentityBasicAuthFilter.scala

```diff
@@ -100,19 +98,34 @@ class RackspaceIdentityBasicAuthFilter @Inject()(configurationService: Configura
   }
 
   override def doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, filterChain: FilterChain) {
-    new FilterLogicHandlerDelegate(servletRequest, servletResponse, filterChain).doFilter(this)
+    val wrappedHttpRequest = new HttpServletRequestWrapper(servletRequest.asInstanceOf[HttpServletRequest])
+    val httpResponse = servletResponse.asInstanceOf[HttpServletResponse]
+
+    val filterAction = handleRequest(wrappedHttpRequest, httpResponse)
+    filterAction match {
```

Comment:
So nope, the Scala compiler does not require an exhaustive match.

If we leave those actions off, and they are used in the future without this code being update, the error the user would see would be something like:
```
scala.MatchError: NOT_SET (of class org.openrepose.commons.utils.servlet.filter.FilterAction)
	at org.openrepose.filters.openstackidentityv3.A$A8$A$A8.get$$instance$$res0(Test.sc0.tmp:5)
	at #worksheet#.#worksheet#(Test.sc0.tmp:18)
```

If we include them here, they won't see an error for PASS, and for NOT_SET they will see "Unexpected internal filter state".

In both cases, a 500 response will be sent for the associated request.

Thoughts/opinion?